corehq/apps/hqwebapp/static/hqwebapp/js/daterangepicker.config.js

```diff
@@ -40,8 +40,18 @@ $.fn.createDateRangePicker = function (
         // Parse dates as local dates to avoid timezone conversion
         var startParts = startdate.split('-');
         var endParts = enddate.split('-');
-        config.startDate = new Date(startParts[0], startParts[1] - 1, startParts[2]);
-        config.endDate = new Date(endParts[0], endParts[1] - 1, endParts[2]);
+
+        config.startDate = new Date(
+            parseInt(startParts[0], 10),
```

Comment:
The second argument shouldn't be required for `parseInt` using base 10, but it's fine that it's here.